Commands/Branding/DisableResponsiveUI.cs

```diff
@@ -7,8 +7,8 @@ namespace SharePointPnP.PowerShell.Commands.Branding
     [Cmdlet(VerbsLifecycle.Disable, "SPOResponsiveUI")]
     [CmdletHelp("Disables the PnP Responsive UI implementation on a classic SharePoint Web", Category = CmdletHelpCategory.Branding)]
     [CmdletExample(Code = "PS:> Disable-SPOResponsiveUI",
-    Remarks = @"If previous enabled, will remove the PnP Responsive UI from a site.",
-    SortOrder = 1)]
+        Remarks = @"If previous enabled, will remove the PnP Responsive UI from a site.",
```

Comment:
Actually before the change it was written wrong. Maybe better to say "If enabled previously, this will remove the PnP Responsive UI from a site."